components/event-processor-manager/org.wso2.carbon.event.processor.manager.core/src/main/java/org/wso2/carbon/event/processor/manager/core/EventManagementService.java

```diff
@@ -35,5 +35,10 @@
 
     public void unregisterEventSync(String syncId);
 
+    public void updateLatestEventSentTime(String publisherName, int tenantId, long timestamp);
+
+    public long getLatestEventSentTime(String publisherName, int tenantId);
+
+    public long getClusterTimeInMilies();
```

Comment:
All methods in an interface are expected to be public and abstract. Therefore, using the access modifiers is redundant.